### DISCUSSION TITLE
Styles for the markdown editor

### DISCUSSION THREAD

**Discussion by fimion** [StaticJsCMS/static-cms#878] [Ideas]
I am currently editing an article and relized that for some reason an inline `<code>` reference doesn't appear to get any sort of formatting, and I'm having a hard time figuring out which words have been highlighted as such. I don't know how easy it is to apply formatting to the editor stuff, but it might be wise to make sure that things have some distinction? 

I might be able to take a crack at this at some point.

**Comment by KaneFreeman** [comment]
This is a bug. I've created https://github.com/StaticJsCMS/static-cms/issues/879 to address it.
